styles/styles.css

```diff
@@ -0,0 +1,662 @@
+@import url("./fonts.css");
+@import url("./all.min.css");
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  max-width: 1264px;
+  padding: 0 16px;
+  margin: 0 auto;
+}
+
+input {
+  border: none;
+}
+
+.phone {
+  display: none;
+}
+
+.trip__background {
+  background-image: url("../images/castelmezzano-1979546 1.jpg");
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
<details>
  <summary>There is a hint for your question from the PR's main comment. Don't open if you wanted advice only. </summary>
  
```
min-height: 100vh;
```
</details>

Reply:
wrote another  background-position